### DISCUSSION TITLE
errors copying somes folder/mails

### DISCUSSION THREAD

**Discussion by spirit72k** [pimalaya/neverest#11] [Q&A]
Hi

I have an error when syncing :

 cannot exec imap action: No Response: Client tried to access nonexistent namespace. (Mailbox name should probably be prefixed with: INBOX.) (0.001 + 0.000 secs)., attempt (1)

I got this error on the majority of the folders from the origin mailbox ( origin mailbox size is ~ 2GB , target size after syncing ~620MB)

An idea anynone?

Thx for help.

**Comment by soywod** [comment]
Hi, which version of the CLI do you run? Could you also share your configuration?
